src/main/java/org/jboss/tools/intellij/analytics/Cli.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.tools.intellij.analytics;
+
+import com.intellij.openapi.util.SystemInfo;
+
+public class Cli {
```

Comment:
You can add cli file path into existing https://github.com/redhat-developer/intellij-dependency-analytics/blob/main/src/main/java/org/jboss/tools/intellij/analytics/Platform.java.

e.g

```
private static final Platform WINDOWS = new Platform("analytics-lsp-win.exe", "crda_0.1.2_Windows_64bit.tar.gz"); 
private static final Platform LINUX = new Platform("analytics-lsp-linux", ..); 
private static final Platform MACOS = new Platform("analytics-lsp-macos");
```

Executable name and version shouldn't part of this. It should be part of the Cli clss